lib/vue/src/components/map/CMap.vue

```diff
@@ -81,28 +108,37 @@ export default {
       default: () => ([])
     },
 
-    localValueIndex:{
+    activeMarkerIndex:{
       type: Number,
       default: undefined
```

Comment:
If its not required, undefined will be the default anyway so this line isn't needed